core/arch/arm/include/mm/mobj.h

```diff
@@ -39,6 +39,7 @@
 struct mobj {
 	const struct mobj_ops *ops;
 	size_t size;
+	size_t granule;
```

Comment:
Ok, thanks. Maybe it could be renamed `phys_granule` for clarity. 

Since this granule is fixed for each 'kind' of memory object, I would prefer this to be provided through the related object operations, Either through a function, or straight from a size_t value in the `struct mobjs_ops`.